format/spec.md

```diff
@@ -607,7 +607,7 @@ A manifest stores files for a single partition spec. When a table’s partition
 
 A manifest file must store the partition spec and other metadata as properties in the Avro file's key-value metadata:
 
-| v1         | v2         | Key                 | Value                                                                        |
+| v1         | v2/v3      | Key                 | Value                                                                        |
```

Comment:
If the columns are identical, merged them with header update because it avoids the effort to figure out what has changed between the versions. Plus we may have v4 and v5 etc which makes tables very big in future if we keep a column for each.

Reply:
We may use `v2+`

Reply:
I think if there's no change the new format version should inherit existing behavior.

Reply:
I prefer v2+ as well 👍